src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,12 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+
+public abstract class Command {
+    protected boolean isExit = false;
+    public boolean isExit() {
+        return this.isExit;
+    }
+    public abstract String execute(TaskList tasks, Storage storage);
```

Comment:
Could use higher level of abstraction here. Maybe have another class to handle UI related methods instead of returning String directly as there might be commands that dont require any output.